src/Pages/Home.js

```diff
@@ -67,10 +67,14 @@ class Home extends Component {
           twitterImage={home.metadataTwitterImage}
         />
         {home.heroUnit &&
-          home.heroUnit.length && <Banner heroUnit={home.heroUnit[0]} />}
+          home.heroUnit.length &&
```

Comment:
Is this necessary?

Reply:
Hi @makma, they are actually not, I have removed it.